bliss/generate.py

```diff
@@ -17,7 +17,7 @@
 
 
 def generate(cfg: DictConfig):
-    file_data_capacity = cfg.cached_simulator.file_data_capacity
+    file_data_capacity = cfg.generate.file_data_capacity
```

Comment:
Could we rename `file_data_capacity` to something like `images_per_file`? (Is that what it represents?) Otherwise it's not clear what the units are (file capacity is often measured in number of bytes).